concepticondata/conceptlists/Simko-2020-205.tsv

```diff
@@ -0,0 +1,206 @@
+Simko-2020-205-156	156	поле	pole	field	concepts concerning nature, tools and housing	212	FIELD
+Simko-2020-205-157	157	река	reka	river	concepts concerning nature, tools and housing	666	RIVER
+Simko-2020-205-158	158	роса	rosa	dew	concepts concerning nature, tools and housing	1977	DEW
+Simko-2020-205-159	159	сняг	snjag	snow	concepts concerning nature, tools and housing	784	SNOW
+Simko-2020-205-160	160	слънце	slănce	sun	concepts concerning nature, tools and housing	1343	SUN
+Simko-2020-205-161	161	вятър	vjatăr	wind	concepts concerning nature, tools and housing	960	WIND
+Simko-2020-205-162	162	вода	voda	water	concepts concerning nature, tools and housing	948	WATER
+Simko-2020-205-163	163	воз	voz	cart load, cart, wagon	concepts concerning nature, tools and housing	122	CART
+Simko-2020-205-164	164	земя	zemja	earth	concepts concerning nature, tools and housing	1228	EARTH (SOIL)
+Simko-2020-205-165	165	злато	zlato	gold	concepts concerning nature, tools and housing	1369	GOLD
+Simko-2020-205-166	166	доя	doja	milk	concepts concerning nourishment	635	MILK
+Simko-2020-205-167	167	ям	jam	eat	concepts concerning nourishment	1336	EAT
+Simko-2020-205-168	168	хляб	hljab	bread	concepts concerning nourishment	1368	BREAD
+Simko-2020-205-169	169	квас	kvas	yeast	concepts concerning nourishment	967	YEAST
+Simko-2020-205-170	170	лой	loj	tallow	concepts concerning nourishment		
+Simko-2020-205-171	171	масло	maslo	fat, grease	concepts concerning nourishment	323	FAT (ORGANIC SUBSTANCE)
+Simko-2020-205-172	172	мед	med	honey	concepts concerning nourishment	942	HONEY
+Simko-2020-205-173	173	мляко	mljako	milk	concepts concerning nourishment	635	MILK
+Simko-2020-205-174	174	месо	meso	meat	concepts concerning nourishment	2615	FLESH OR MEAT
+Simko-2020-205-175	175	пия	pija	drink	concepts concerning nourishment	731	BEVERAGE
+Simko-2020-205-176	176	пиво	pivo	beer	concepts concerning nourishment	1639	BEER
+Simko-2020-205-177	177	сало	salo	fat	concepts concerning nourishment	323	FAT (ORGANIC SUBSTANCE)
+Simko-2020-205-178	178	сирене	sirene	(white) cheese	concepts concerning nourishment	1569	CHEESE
+Simko-2020-205-179	179	сит	sit	fed	concepts concerning nourishment		
+Simko-2020-205-180	180	тесто	testo	dough	concepts concerning nourishment	273	DOUGH
+Simko-2020-205-181	181	вино	vino	wine	concepts concerning nourishment	1524	WINE
+Simko-2020-205-182	182	брада	brada	beard	body parts	678	BEARD
+Simko-2020-205-183	183	чело	čelo	forehead	body parts	123	FOREHEAD
+Simko-2020-205-184	184	челюст	čeljust	jaw	body parts	798	JAW
+Simko-2020-205-185	185	черво	červo	gut	body parts	1334	GUTS
+Simko-2020-205-186	186	длан	dlan	palm	body parts	1183	PALM OF HAND
+Simko-2020-205-187	187	глава	glava	head	body parts	1256	HEAD
+Simko-2020-205-188	188	език	ezik	tongue	body parts	1205	TONGUE
+Simko-2020-205-189	189	кост	kost	bone	body parts	1394	BONE
+Simko-2020-205-190	190	кожа	koža	skin	body parts	2613	SKIN (HUMAN)
```

Comment:
It's not clearly distinguished so I'd map it to the overarching concept 763 SKIN.